teps/0079-tekton-catalog-support-tiers.md

```diff
@@ -351,21 +351,24 @@ it may be difficult to identify which Tekton Catalog it came from
 because they won't have the labels added by the CLI.
 
 To make it easy for users to identify the source Catalog, we propose 
-adding a `tekton.dev/catalog` annotation to the resource yaml file, 
-as such:
+adding two annotations:
+* `tekton.dev/catalog` with the three part domain unique identifier
```

Comment:
is it possible to provide more details here on what this domain unique identifier is?

the description of this PR says "In this change, we update the tekton.dev/catalog annotation
to use the URL of the Catalog instead of its name" but i think in response to @chmouel 's comment (https://github.com/tektoncd/community/pull/613#issuecomment-1022954361) this was expanded to include this as well, would be great to explain in detail what a user should expect to see here and what folks should put here for their own catalogs

Reply:
@bobcatfish thanks for pointing this out. To discuss more on this I have already started a [thread on slack](https://tektoncd.slack.com/archives/CK9JGKS20/p1643632267472469). If everyone agrees then I can start a thread here as well to get some more thoughts around it.

Reply:
may be better to have conversation on github isnt it ?

The idea behind three dot domain id : 

* URL can changes and can be moved elsewhere, if we want to know the provenance of a catalog task the url is not necessary something we can rely on
* domain id allows to easily know which provider providing this catalog. A company may want to introduce their own catalogs for their users and having a domain id make sure there would be no conflicts with official always.
 * Tools can always rely on a the catalog id to be the same

Reply:
1. `<support-tier>.<org>.catalog`. Example -> official.tekton.catalog, official.foobar.catalog, community.tekton.catalog, etc
2. `<catalog-repo-name>.<org>.catalog` Example -> official-catalog.tekton.catalog, community-catalog.tekton.catalog, etc
3. Organization specific domain uri. example -> `com.google.gke` or `com.redhat.openshift` etc
4. Any other suggestions?

cc @jerop @vdemeester @bobcatfish @afrittoli @chmouel

Reply:
I vote for 3

Reply:
I would vote for 3 too

Reply:
3 sgtm!

Reply:
Thanks for the discussion! Updated the TEP, please take a look :)